src/handlers/shared/check-unassigns.ts

```diff
@@ -0,0 +1,20 @@
+import { Context } from "../../types";
+
+export async function wasPreviouslyUnassigned(
+  context: Context,
+  senderLogin: Context["payload"]["sender"],
+  issueNumber: Context["payload"]["issue"]
+): Promise<boolean> {
+  try {
+    const issueEvents = await context.octokit.issues.listEventsForTimeline({
+      owner: context.payload.repository.owner.login,
+      repo: context.payload.repository.name,
+      issue_number: issueNumber.id,
+    });
+
+    return issueEvents.data.some((event) => event.event === "unassigned" && event.event === senderLogin);
```

Comment:
I'm sceptical that `event.event` can be both `unassigned` and `senderLogin`, QA would be ideal

Reply:
Just noticed that this fails on the test CI. I suppose I should be more cautious of reviews from mobile as it is not easy to see these details sometimes.

Reply:
Just noticed that this fails on the test CI. I suppose I should be more cautious of reviews from mobile as it is not easy to see these details sometimes.